src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -318,10 +318,25 @@ template <bool UseDgSubcell,
           typename InitialData, typename... InterpolationTargetTags>
 struct GhValenciaDivCleanTemplateBase<
     EvolutionMetavarsDerived<InitialData, InterpolationTargetTags...>,
-    UseDgSubcell> : public virtual GhValenciaDivCleanDefaults<UseDgSubcell> {
+    UseDgSubcell>
+    : public virtual GhValenciaDivCleanDefaults<
+          UseDgSubcell, evolution::is_numeric_initial_data_v<InitialData>> {
   using derived_metavars =
       EvolutionMetavarsDerived<InitialData, InterpolationTargetTags...>;
-  using defaults = GhValenciaDivCleanDefaults<UseDgSubcell>;
+
+  static constexpr bool use_dg_subcell = UseDgSubcell;
```

Comment:
I don't understand why any of the changes in this commit are happening. This does more than the template parameter. Also, in a couple commits I ask if we can check locally if the analytic tag is retrievable, and if not then we assume numeric initial data. I think if that's possible then we can skip this commit completely.

Reply:
dropped this commit